modules/connectivity/locals.tf

```
@@ -831,13 +832,31 @@ locals {
     }
     "${local.root_id}-corp" = {
       parameters = {
-        Enable-DDoS-VNET = {
-          ddosPlan = local.ddos_protection_plan_resource_id
+        Deploy-Private-DNS-Zones = {
+          azureFilePrivateDnsZoneId                     = "${local.private_dns_zone_prefix}privatelink.afs.azure.net"
+          azureWebPrivateDnsZoneId                      = "${local.private_dns_zone_prefix}privatelink.webpubsub.azure.com"
+          azureBatchPrivateDnsZoneId                    = "${local.private_dns_zone_prefix}privatelink.${local.location}.batch.azure.com"
+          azureAppPrivateDnsZoneId                      = "${local.private_dns_zone_prefix}privatelink.azconfig.io"
+          azureAsrPrivateDnsZoneId                      = "${local.private_dns_zone_prefix}${local.location}.privatelink.siterecovery.windowsazure.com"
```

```suggestion
          azureAsrPrivateDnsZoneId                      = "${local.private_dns_zone_prefix}privatelink.siterecovery.windowsazure.com"
```

From testing with others, the location geo code is added at the 'A' record creation so itsnt required on the zone itself: as per https://docs.microsoft.com/en-us/azure/site-recovery/azure-to-azure-how-to-enable-replication-private-endpoints